src/gui/Assignments.js

```diff
@@ -0,0 +1,36 @@
+
+/* global document */
+
+class Assignment {
+
+    constructor(name, link, dueDate) {
+        this.aName = name;
+        this.aLink = link;
+        this.dueDate = dueDate;
+    }
+}
+
+const assignments = [
+    new Assignment("Deliverable 2", "", new Date(2019, 10, 4)),
```

Comment:
Not sure if this should be taken from the DB for your US or not.... I won't mark it against you for now since I don't see any explicit requirement to do so.